src/views/download/download.jsx

```diff
@@ -0,0 +1,171 @@
+                                <p><FormattedMessage id='download.supportMaterialsBody' /></p>
+                                <p>
+                                    <FormattedMessage id='download.starterProjects' /> -
+                                    {' '}<a href="https://scratch.mit.edu/scratchr2/static/sa/Scratch2StarterProjects.zip">
+                                        <FormattedMessage id='download.download' />
+                                    </a>
+                                </p>
+                                <p>
+                                    <FormattedMessage id='download.gettingStarted' /> -
+                                    {' '}<a href="https://cdn.scratch.mit.edu/scratchr2/static/__709da8e5f3d72129538a4ccdbcbf5f2a__/pdfs/help/Getting-Started-Guide-Scratch2.pdf">
+                                        <FormattedMessage id='download.download' />
+                                    </a>
+                                </p>
+                                <p>
+                                    <FormattedMessage id='download.scratchCards' /> -
+                                    {' '}<a href="https://cdn.scratch.mit.edu/scratchr2/static/__709da8e5f3d72129538a4ccdbcbf5f2a__/pdfs/help/Scratch2Cards.pdf">
+                                        <FormattedMessage id='download.download' />
+                                    </a>
+                                </p>
+                            </div>
+                        </FlexRow>
+                    </section>
+                    
+                    <section id="updates">
+                        <span className="nav-spacer"></span>
+                        <h2><FormattedMessage id='download.updatesTitle' /></h2>
+                        <p><FormattedMessage id='download.updatesBody' /></p>
+                        <p><FormattedMessage id='download.currentVersion' /></p>
+                    </section>
+                    
+                    <section id="other">
+                        <span className="nav-spacer"></span>
+                        <h2><FormattedMessage id='download.otherVersionsTitle' /></h2>
+                        <p><FormattedHTMLMessage id='download.otherVersionsOlder' /></p>
+                        <p><FormattedHTMLMessage id='download.otherVersionsAdmin' /></p>
+                    </section>
+                    
+                    <section id="issues">
+                        <span className="nav-spacer"></span>
+                        <h2><FormattedMessage id='download.knownIssuesTitle' /></h2>
+                        <p><FormattedMessage id='download.knownIssuesOne' /></p>
+                        <p><FormattedMessage id='download.knownIssuesTwo' /></p>
+                        <p><FormattedHTMLMessage id='download.knownIssuesThree' /></p>
+                        <a href="https://scratch.mit.edu/discuss/3/">
```

Comment:
In the existing page, this appears as a button – thoughts on having it show up as a button here too? In that case, you can just require the `button.scss` file, and give it the button class here